tinywl/build.zig

```diff
@@ -29,6 +29,17 @@ pub fn build(b: *Builder) void {
         .dependencies = &[_]Pkg{ wayland, xkbcommon, pixman },
     };
 
+    // These must be manually kept in sync with the versions wlroots supports
+    // until wlroots gives us the option request a specific version.
```

Comment:
Typo inherited from river :)

```suggestion
    // until wlroots gives us the option to request a specific version.
```